src/me/hkgumbs/tictactoe/test/java/simulation/RulesConfigurationTest.java

```diff
@@ -0,0 +1,31 @@
+package me.hkgumbs.tictactoe.test.java.simulation;
+
+import me.hkgumbs.tictactoe.main.java.board.SquareBoard;
+import me.hkgumbs.tictactoe.main.java.simulation.*;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class RulesConfigurationTest {
+    @Test
+    public void defaultRules() {
+        Simulation simulation = new DefaultSimulation();
+        OutputStream outputStream = new ByteArrayOutputStream();
+        Configuration[] configurations = new Configuration[]{
+                new SizeConfiguration(),
+                new RulesConfiguration(outputStream)
+        };
+        try {
+            for (Configuration configuration : configurations)
+                configuration.apply(Arrays.asList(), simulation);
+        } catch (Configuration.CannotApplyException e) {
+            e.printStackTrace();
```

Comment:
What should happen in a test run if this exception gets thrown? Should the test fail? The way this is written, we'd see a stack trace, but no test would fail.